linkerd/app/src/env.rs

```diff
@@ -187,11 +187,29 @@ pub const ENV_INBOUND_IPS: &str = "LINKERD2_PROXY_INBOUND_IPS";
 pub const ENV_IDENTITY_DISABLED: &str = "LINKERD2_PROXY_IDENTITY_DISABLED";
 pub const ENV_IDENTITY_DIR: &str = "LINKERD2_PROXY_IDENTITY_DIR";
 pub const ENV_IDENTITY_TRUST_ANCHORS: &str = "LINKERD2_PROXY_IDENTITY_TRUST_ANCHORS";
-pub const ENV_IDENTITY_IDENTITY_LOCAL_NAME: &str = "LINKERD2_PROXY_IDENTITY_LOCAL_NAME";
 pub const ENV_IDENTITY_TOKEN_FILE: &str = "LINKERD2_PROXY_IDENTITY_TOKEN_FILE";
 pub const ENV_IDENTITY_MIN_REFRESH: &str = "LINKERD2_PROXY_IDENTITY_MIN_REFRESH";
 pub const ENV_IDENTITY_MAX_REFRESH: &str = "LINKERD2_PROXY_IDENTITY_MAX_REFRESH";
 
+// This config is here for backwards compatibility. If set, both the tls id and the server
+// name will be set to the value specified in this config. The values needs to be a DNS
+// name
+pub const ENV_IDENTITY_IDENTITY_LOCAL_NAME: &str = "LINKERD2_PROXY_IDENTITY_LOCAL_NAME";
+// Configures the TLS Id of the proxy inbound server. The value is expected to match the
+// DNS or URI SAN of the leaf certificate that will be provisioned to this proxy.
+pub const ENV_IDENTITY_IDENTITY_SERVER_ID: &str = "LINKERD2_PROXY_IDENTITY_SERVER_ID";
+// Configures the server name of this proxy. This value is expected to match the value
+// that clients include in the SNI extension of the ClientHello, whenever they try to
+// establish a TLS connection that shall be terminated by this proxy
+pub const ENV_IDENTITY_IDENTITY_SERVER_NAME: &str = "LINKERD2_PROXY_IDENTITY_SERVER_NAME";
+
+// If this config is set, then the proxy will be configured to use Spire as identity
+// provider
+pub const ENV_IDENTITY_SPIRE_SOCKET: &str = "LINKERD2_PROXY_IDENTITY_SPIRE_SOCKET";
+pub const IDENTITY_SPIRE_BASE: &str = "LINKERD2_PROXY_IDENTITY_SPIRE";
+const DEFAULT_SPIRE_BACKOFF: ExponentialBackoff =
+    ExponentialBackoff::new_unchecked(Duration::from_millis(1), Duration::from_millis(5), 0.1);
```

Comment:
```suggestion
    ExponentialBackoff::new_unchecked(Duration::from_millis(100), Duration::from_secs(1), 0.1);
```

This seems a bit hot-loopy to me. If we need to backoff, let's give the OS some time to catch up and get Spire into a better state.

Reply:
Yes this was copy paste. I was thinking of changing it to something like 1 - 5 secs, but what you suggest also makes sense.